backend/seeder/main.go

```diff
@@ -67,10 +70,17 @@ func seedTestData(db *gorm.DB) {
 		log.Printf("Failed to unmarshal test data: %v", err)
 	}
 	for idx, u := range users {
+		u.Password = "ChangeMe!"
```

Comment:
Previously, users created by seeding the database wouldn't delete properly, because they didn't exist in kratos

Reply:
This, along with `srv.Db.DeleteUser(id)` in user delete handler will fix the delete user bug, right?